caff-backend/src/main/java/com/example/cb/repository/CaffRepository.java

```diff
@@ -7,4 +7,5 @@
 
 @Repository
 public interface CaffRepository extends JpaRepository<Caff, Long> {
```

Comment:
Ha a repository a CrudRepositoryból öröklődik, akkor tartalmazza a findById függvényt alapból :) 
Egyelőre jó lesz, majd átírom, mert valószítű akkor több helyen is JpaRepository az ős, csak hogy lásd, hogy ilyen is van :)